docs/doc/11-integrations/00-api/02-clickhouse-handler.md

```diff
@@ -9,7 +9,7 @@ description:
 
 ## Overview
 
-Databend is ClickHouse wire protocol-compatible, allow you to connect to Databend server with Clickhouse client, make it easier for users/developers to use Databend.
+Databend is ClickHouse HTTP API wire protocol-compatible, allow you to connect to Databend server with ClickHouse HTTP Handler, make it easier for users/developers to use Databend.
```

Comment:
Databend is ClickHouse HTTP API wire protocol-compatible, allowing you to connect to Databend with ClickHouse HTTP Handler.